integration/fuse/src/test/java/alluxio/fuse/AlluxioFuseFileSystemTest.java

```diff
@@ -76,6 +76,28 @@ public void before() throws Exception {
   }
 
   @Test
+  public void chmod() throws Exception {
+    long mode = 123;
+    mFuseFs.chmod("/foo/bar", mode);
+    AlluxioURI expectedPath = BASE_EXPECTED_URI.join("/foo/bar");
+    SetAttributeOptions options = SetAttributeOptions.defaults().setMode(new Mode((short) mode));
+    verify(mFileSystem).setAttribute(expectedPath, options);
+  }
+
+  @Test
+  public void chown() throws Exception {
+    long[] uidGid = AlluxioFuseUtils.getUidAndGid();
+    mFuseFs.chown("/foo/bar", uidGid[0], uidGid[1]);
```

Comment:
could we use hardcoded strings for the user and group? Ideally we wouldn't use the same code for both the test and implementation

Reply:
I understand. The difficulty is that the user and group must exist in the local machine, unless we mock `getUidAndGid `